src/main/java/simplexity/adminhax/commands/basic/RenameItem.java

```diff
@@ -40,7 +40,7 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
         }
         ItemStack heldItem = player.getInventory().getItemInMainHand();
         if (heldItem.isEmpty() || heldItem.getType().isEmpty()) {
```

Comment:
Since you are already touching this file but you are not editing this line, this is up to you...
```suggestion
        if (heldItem.isEmpty()) {
```
According to [ItemStack.isEmpty()](https://jd.papermc.io/paper/1.21.8/org/bukkit/inventory/ItemStack.html#isEmpty())...

> Returns whether this item stack is empty and contains no item. This means it is either **_air_** or the stack has a size of 0.

According to [Material.isEmpty()](https://jd.papermc.io/paper/1.21.8/org/bukkit/Material.html#isEmpty())...

> If the type is either AIR, CAVE_AIR or VOID_AIR

So calling both `ItemStack.isEmpty()` and `Material.isEmpty()` is redundant.
`Material.isEmpty()` has also been deprecated since 1.21.5 and was replaced with `Material.isAir()`.